src/app/admin/admin.component.ts

```diff
@@ -1,21 +1,98 @@
 import { Component, OnInit, OnDestroy } from '@angular/core';
 import { NavbarService } from 'app/core/navbar.service';
+import { Subject } from 'rxjs';
+import { ActivatedRoute, Router } from '@angular/router';
+import { takeUntil, skipWhile, take } from 'rxjs/operators';
+import { AuthService } from 'app/core/auth.service';
+import { CollectionService, Collection } from 'app/core/collection.service';
+
 
 @Component({
   selector: 'clark-admin',
   templateUrl: './admin.component.html',
   styleUrls: ['./admin.component.scss']
 })
 export class AdminComponent implements OnInit, OnDestroy {
+  destroyed$: Subject<void> = new Subject();
+
+  authorizedCollections: Collection[] = [];
```

Comment:
Doesn't this variable name imply that there are unauthorized collections that this component is in some way concerned about?

Reply:
I would argue that, since this is the root level component and it is responsible for retrieving the collections that the currently logged in user is authorized to view in this context, this variable name makes sense?

Reply:
Yeah, I can get behind that.

I think my uneasiness is perhaps around the fact that this is the `AdminComponent` yet it is being used by both admin and curator user types. While curators are only authorized for a subset of collections, admins should always be authorized for all collections. How do you feel about renaming this component?

You mentioned it is the root level component, would that make CollectionDashboardComponent a better name?

Reply:
Yeah that probably makes more sense. What do you think we should do about the route? This currently lives at /admin, I feel collection-dashboard is a bit verbose?